vtm-ios-example/src/org/oscim/ios/test/ExampleLauncher.java

```diff
@@ -38,11 +38,12 @@ protected IOSApplication createApplication() {
         float scale = (float) (getIosVersion() >= 8 ? UIScreen.getMainScreen().getNativeScale() : UIScreen.getMainScreen().getScale());
         CanvasAdapter.dpi *= scale;
 
-        IOSMapApp iosMapApp = new IOSMapApp();
-//        IOSLineTexTest iosMapApp = new IOSLineTexTest();
-//        IOSMapAppCluster iosMapApp = new IOSMapAppCluster();
+//        IOSMapApp iosMapApp = new IOSMapApp();
+//        IOSPathLayerTest iosMapApp = new IOSPathLayerTest();
+        IOSLineTexBucketTest iosMapApp = new IOSLineTexBucketTest();
```

Comment:
It'd be nice if you could add some initial iOS screen with menu buttons to select each sample, like in Android?